crates/hyperswitch_domain_models/Cargo.toml

```diff
@@ -14,11 +14,17 @@ payouts = ["api_models/payouts"]
 
 [dependencies]
 # First party deps
-api_models = { version = "0.1.0", path = "../api_models" }
+api_models = { version = "0.1.0", path = "../api_models", features = ["errors"] }
 common_enums = { version = "0.1.0", path = "../common_enums" }
 common_utils = { version = "0.1.0", path = "../common_utils" }
 masking = { version = "0.1.0", path = "../masking" }
 diesel_models = { version = "0.1.0", path = "../diesel_models", features = ["kv_store"] }
+cards = {version = "0.1.0", path = "../cards"}
+router_derive = {version = "0.1.0", path = "../router_derive"}
+mime = "0.3.17"
+url = { version = "2.5.0", features = ["serde"] }
+utoipa = { version = "4.2.0", features = ["preserve_order", "preserve_path_order", "time"] }
+serde_with = "3.7.0"
```

Comment:
Please segregate (and later, sort) them as first and third party dependencies correctly: `mime`, `serde_with`, `url` and `utoipa` are third party depndencies.